libs/login/feature/src/lib/login.component.ts

```diff
@@ -149,29 +149,34 @@ export class LoginComponent implements OnInit {
       )
       else
       {
+        //this.presentLoadingWithOptions();
```

Comment:
Commented Out Code

Reply:
Gonna leave it there, This is how we will call the other loader